### ISSUE TITLE
SUPPORTED_RUNTIMES silently rewrites llama.cpp and transformers to ollama

### ISSUE THREAD

**Issue by signerless** [Pavelevich/llm-checker#123]
`--runtime` is advertised on several commands as
`auto|ollama|vllm|mlx|llama.cpp|transformers`, but the runtime module only knows
three of them:

`src/runtime/runtime-support.js:1`

```js
const SUPPORTED_RUNTIMES = ['ollama', 'vllm', 'mlx'];
```

`normalizeRuntime()` falls back to `'ollama'` for anything not in that list, so:

```
normalizeRuntime('llama.cpp')     -> 'ollama'
normalizeRuntime('transformers')  -> 'ollama'
normalizeRuntime('auto')          -> 'ollama'
```

That means `getRuntimeInstallCommand()`, `getRuntimePullCommand()` and
`getRuntimeRunCommand()` all emit Ollama commands for a user who asked for
llama.cpp — a GGUF workflow gets told to run `ollama pull`.

The registry half of the codebase does model these runtimes correctly
(`src/data/registry-recommender.js` routes GGUF to llama.cpp), so the two halves
disagree.

### Suggested fix

Widen `SUPPORTED_RUNTIMES` to match what the CLI advertises and give each
runtime its own install/pull/run commands, or narrow the `--help` text. Either
way `normalizeRuntime` should not silently rewrite an unknown runtime to a
different one — returning `null` and letting the caller decide is safer.

**Comment by signerless** [comment]
Resuelto y fusionado en main mediante #135.

`llama.cpp`, `transformers` y `auto` conservan su selección; los runtimes desconocidos devuelven null y los comandos de CLI rechazan la entrada inválida. Se generan comandos propios de cada runtime y se evita inventar un repositorio o un archivo GGUF a partir de un tag de Ollama.

Validación real: ejecutados los comandos generados de descarga y generación de 64 tokens con llama.cpp (build 10678, stories260K.gguf) y Transformers (Python 3.12, Transformers 5.16.1, Torch 2.14.0 CPU, HuggingFaceTB/SmolLM2-135M). Ambos terminaron con código 0. La descarga de Transformers tardó 99,5 s y su generación 9,4 s. Suite local: 61/61; CI completa en Node 18/20/22, Windows y policy gate en verde.
